phoenix-core/src/test/java/org/apache/phoenix/parse/PhoenixRowTimestampFunctionTest.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.phoenix.parse;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+public class PhoenixRowTimestampFunctionTest {
+
+    @Test
+    public void testExpressionWithPhoenixRowTimestamp() throws Exception {
+        ParseNode parseNode = SQLParser.parseCondition("(PHOENIX_ROW_TIMESTAMP() = PK2)");
+        boolean hasPhoenixRowTimestampParseNode = false;
+        for (ParseNode childNode : parseNode.getChildren()) {
+            if (childNode.getClass().isAssignableFrom(PhoenixRowTimestampParseNode.class)) {
+                assertEquals("", childNode.getChildren().size(), 0);
```

Comment:
nit: Remove empty string print in assertion and also fix if condition formatting